sql-jdbc/src/main/java/org/opensearch/jdbc/ResultSetImpl.java

```diff
@@ -1071,14 +1075,22 @@ public Date getDate(String columnLabel, Calendar cal) throws SQLException {
 
     @Override
     public Time getTime(int columnIndex, Calendar cal) throws SQLException {
-        // TODO - implement?
-        return null;
+        log.debug(() -> logEntry("getTime (%d, %s)", columnIndex,
+                cal == null ? "null" : "Calendar TZ= " + cal.getTimeZone()));
+        checkCursorOperationPossible();
+        Time value = getTimeX(columnIndex, cal);
```

Comment:
this will get called with or without debug.  Can we put this in the log.debug() statement below? 
```
log.debug(() -> logExit("getTime", getTimeX(columnIndex, cal)));
```